### PR TITLE
Upgrade Dependencies

### PR DESCRIPTION
**Pull request by nitro-marky**
This PR:

- Updates dependencies
- Removes Helm charts
- Updates check version and release flow
- Adds c8

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -35,84 +35,13 @@ jobs:
...
     name: Run lint
     runs-on: ubuntu-latest
     needs: [preconditions]
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@master
+      - uses: actions/setup-node@v3
         with:
           node-version: 16.x
       - name: Use npm v8
```
Comment (dblane-digicatapult) [inline .github/workflows/release.yml:47]:
```suggestion
     - name: Use npm latest
        run: npm install -g npm@latest
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -192,3 +121,34 @@ jobs:
...
         run: |
           exit $([[ "${{steps.get_version.outputs.IS_NEW_VERSION}}" = "true" ]] && echo 0 || echo 1)
+
+  docker-build:
+    name: 'Build docker image'
+    needs:
+      - lint
+      - tests
+      - check-version
+      - dependency-check
```
Comment (dblane-digicatapult) [inline .github/workflows/tests.yml:131]:
```suggestion
```

Comment (dblane-digicatapult) [reply .github/workflows/tests.yml:131]: We want this job to run in parallel with the others.

File: Dockerfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# syntax=docker/dockerfile:1.0.0-experimental
+# syntax=docker/dockerfile:1.0.1-experimental
```
Comment (dblane-digicatapult) [inline Dockerfile:1]:
```suggestion
# syntax=docker/dockerfile:1.5
```

File: Dockerfile [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
 
-FROM node:16-alpine AS build
+FROM node:lts-alpine3.17 AS build
```
Comment (dblane-digicatapult) [inline Dockerfile:3]:
```suggestion
FROM node:lts-alpine AS build
```

File: Dockerfile [hunk 5]
```diff
@@ -14,14 +14,14 @@ RUN apk update && \
...
-RUN npm -g install npm@8.x.x
+RUN npm i -g npm@latest
 
 COPY . .
 RUN npm ci --production
 
 ##################################################################################################
 
-FROM node:16-alpine AS runtime
+FROM node:lts-alpine3.17 AS runtime
```
Comment (dblane-digicatapult) [inline Dockerfile:24]:
```suggestion
FROM node:lts-alpine AS runtime
```

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (dblane-digicatapult) [reply .github/workflows/release.yml:47]: Also remove the underlying `run command`

File: .github/workflows/tests.yml [hunk 6]
```diff
@@ -192,3 +121,29 @@ jobs:
...
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: all
+      - name: Setup Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          buildkitd-flags: '--debug'
+      - name: Publish multi-arch image
```
Comment (dblane-digicatapult) [inline .github/workflows/tests.yml:142]:
```suggestion
      - name: Build multi-arch image
```
We aren't publishing.

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

**Review by dblane-digicatapult — APPROVED** [review]
Looks good to me, wait for @jonmattgray or @n3op2 to review before merging.

**Review by jonmattgray — APPROVED** [review]

**Pull request merged**